WordPress/Classes/Models/BloggingPromptSettingsReminderDays+CoreDataClass.swift

```diff
@@ -14,4 +14,21 @@ public class BloggingPromptSettingsReminderDays: NSManagedObject {
         self.sunday = remoteReminderDays.sunday
     }
 
+    func getActiveWeekdays() -> [BloggingRemindersScheduler.Weekday] {
+        return [
+            sunday,
+            monday,
+            tuesday,
+            wednesday,
+            thursday,
+            friday,
+            saturday
+        ].enumerated().flatMap { (index: Int, isReminderActive: Bool) in
```

Comment:
Let's make Xcode happy 😄  (`flatMap` is deprecated)

```suggestion
        ].enumerated().compactMap { (index: Int, isReminderActive: Bool) in
```